src/SumoCoders/FrameworkCoreBundle/Resources/views/empty.html.twig

```diff
@@ -23,19 +23,14 @@
     <![endif]-->
 
     {% block stylesheets %}
-        {# Load full css if browser is > IE9 or not IE9 #}
-        <!--[if gt IE 9]>
-            <link rel="stylesheet" href="/assets/css/style.css" />
-        <![endif]-->
-        <!--[if !IE]> -->
-            <link rel="stylesheet" href="/assets/css/style.css" />
-        <!-- <![endif]-->
-
-        {# Load css and bootstrap seperately if <= IE9 #}
-        <!--[if lte IE 9]>
-            <link rel="stylesheet" href="/assets/css/bootstrap.css" />
-            <link rel="stylesheet" href="/assets/css/style_ie.css" />
-        <![endif]-->
+            {% stylesheets
+                'assets/vendor/select2/select2.css'
```

Comment:
waarom is select2 nodig op de login page? Daar zal toch nooit een dropdown op staan?